webapp/src/components/expanded_view/component.tsx

```diff
@@ -8,13 +8,18 @@ import {OverlayTrigger, Tooltip} from 'react-bootstrap';
 import data from '@emoji-mart/data';
 import Picker from '@emoji-mart/react';
 
+import {Client4} from 'mattermost-redux/client';
+import {getEmojiImageUrl} from 'mattermost-redux/utils/emoji_utils';
```

Comment:
Nice. This will make future migration to the mattermost emoji component much easier